### PR TITLE
Fix rate limits for content votes, add hourly limit for minuses.

### PR DESCRIPTION
**Pull request by Aivean**
Fix: correct key calculation.

Current limits:
* 100 per day for all content votes.
* 30 per hour for minuses.

### CODE REVIEW

File: backend/src/api/VoteController.ts [hunk 1]
```diff
@@ -55,7 +51,7 @@ export default class VoteController {
             id: Joi.number().required()
         });
 
-        this.router.post('/vote/set', this.voteRateLimiter, validate(voteSchema), this.voteDayRateLimiter, (req, res) => this.setVote(req, res));
+        this.router.post('/vote/set', this.voteRateLimiter, validate(voteSchema), (req, res) => this.setVote(req, res));
```
Comment (4vanger) [inline backend/src/api/VoteController.ts:54]: nit
```suggestion
        this.router.post('/vote/set', this.voteRateLimiter, validate(voteSchema), this.setVote);
```

File: backend/src/api/VoteController.ts [hunk 2]
```diff
@@ -76,6 +72,19 @@ export default class VoteController {
             const max = type === 'user' ? 2 : 1;
             const rangedVote = Math.max(Math.min(vote, max), -max);
 
+            const targetUserId = await this.voteManager.getUserIdByVote(id, type);
+            if (targetUserId && type !== 'user') {
+                const voteKey = `${userId}:${targetUserId}`;
+                try {
+                    await this.voteDailyRateLimiter.consume(voteKey);
+                    if (rangedVote < 0) {
```
Comment (4vanger) [inline backend/src/api/VoteController.ts:80]: But why only for minuses? mass-plusing will have same effect on regulation

Comment (Aivean) [reply backend/src/api/VoteController.ts:80]: > mass-plusing will have same effect on regulation

You mean zero effect?

File: backend/src/api/VoteController.ts (hunk 1 above)
Comment (Aivean) [reply backend/src/api/VoteController.ts:54]: Are you sure that `this` will remain properly bound? I'd prefer not to change this blindly without testing.

File: backend/src/api/VoteController.ts (hunk 2 above)
Comment (4vanger) [reply backend/src/api/VoteController.ts:80]: Yup - so why treat them differently

File: backend/src/api/VoteController.ts (hunk 1 above)
Comment (4vanger) [reply backend/src/api/VoteController.ts:54]: No, it wouldn't
```
this.setVote.bind(this)
```
but it is not that slick. Ignore.

**Review by 4vanger — APPROVED** [review]

File: backend/src/api/VoteController.ts (hunk 2 above)
Comment (Aivean) [reply backend/src/api/VoteController.ts:80]: Solely because people are using them differently and complaining only about minuses.

**Pull request merged**